tests/test_wave_03.py

```diff
@@ -0,0 +1,782 @@
+from app.models.video import Video
+from app.models.customer import Customer
+
+VIDEO_1_TITLE = "A Brand New Video"
+VIDEO_1_ID = 1
+VIDEO_1_INVENTORY = 1
+VIDEO_1_RELEASE_DATE = "01-01-2001"
+
+VIDEO_2_TITLE = "Video Two"
+VIDEO_2_ID = 2
+VIDEO_2_INVENTORY = 1
+VIDEO_2_RELEASE_DATE = "12-31-2000"
+
+VIDEO_3_TITLE = "Video Three"
+VIDEO_3_ID = 3
+VIDEO_3_INVENTORY = 1
+VIDEO_3_RELEASE_DATE = "01-02-2001"
+
+CUSTOMER_1_NAME = "A Brand New Customer"
+CUSTOMER_1_ID = 1
+CUSTOMER_1_POSTAL_CODE = "12345"
+CUSTOMER_1_PHONE = "123-123-1234"
+
+CUSTOMER_2_NAME = "Second Customer"
+CUSTOMER_2_ID = 2
+CUSTOMER_2_POSTAL_CODE = "12345"
+CUSTOMER_2_PHONE = "234-234-2345"
+
+CUSTOMER_3_NAME = "Customer Three"
+CUSTOMER_3_ID = 3
+CUSTOMER_3_POSTAL_CODE = "12344"
+CUSTOMER_3_PHONE = "000-000-0000"
+
+
+def test_get_customers_no_query_params_sorts_by_id(client, one_customer, second_customer, third_customer):
```

Comment:
I'm wondering how the arguments passed into these test functions are used. In the IDE they are all greyed out indicating that they aren't in use. What is the effect of having them here? 

Sorry if I'm missing something here, I haven't done DB testing in a longgg while and don't have experience with SQLAlchemy.

Reply:
This is how the test fixtures defined in conftest.py pass the created objects/data to a test, I'm not sure if there's a good syntax highlighting extension that accounts for this situation. Typically `client` is the only parameter we use to make calls in the test, the others just ensure the test data from those conftest functions is present.

Reply:
Thank you for the clarification here!!